src/turbulence_closures/constant_diffusivity_closures.jl

```diff
@@ -3,14 +3,20 @@
 #####
 
 """
-    ConstantIsotropicDiffusivity(T=Float64; ν=1e-6, κ=1e-7)
+    ConstantIsotropicDiffusivity{T}
 
-or
+    ConstantIsotropicDiffusivity(; ν, κ)
 
-    MolecularDiffusivity(T=Float64; ν=1e-6, κ=1e-7)
+Use a constant isotropic turbulent diffusivty closure with constant viscosity `ν` and
+constant thermal diffusivity `κ`.
 
-Return a `ConstantIsotropicDiffusivity` closure object of type `T` with
-viscosity `ν` and scalar diffusivity `κ`.
+Note that this corresponds to the use an eddy viscosity model with constant `ν` and ``κ`.
```

Comment:
How about instead of saying this (which I find slightly obscure), we change the first sentence to

```
Returns parameters for a constant isotropic diffusivity closure with constant viscosity `ν` and constant thermal diffusivity `κ`. `ν` and `κ` may represent molecular diffusivities or turbulent eddy diffusivities.
```

PS I don't think we should claim that instantiating a closure amounts to 'using' something. Instantiating a closure just... well... instantiates a struct of parameters. 'Using' implies to me that more happens than the mere creation of a simple object.